instances/unitOfMeasurement/hertz.jsonld

```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/hertz",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "Hertz is the unit of frequency in the International System of Units (SI), equivalent to one event (or cycle) per second.",
```

Comment:
The hertz is ... AND please add the source of that definition (here is is Wikipedia).